modules/nextflow/src/main/groovy/nextflow/cli/CmdRun.groovy

```diff
@@ -372,6 +376,21 @@ class CmdRun extends CmdBase implements HubOptions {
         runner.execute(scriptArgs, this.entryName)
     }
 
+    protected void printBanner() {
+        if( launcher.options.ansiLog ){
+            log.debug "N E X T F L O W  ~  version ${BuildInfo.version}"
+
+            def fmt = ansi()
+            fmt.a("\n")
+            fmt.bg(Color.CYAN).fg(Color.BLACK).bold().a(" N E X T F L O W ").reset()
```

Comment:
@pditommaso this isn't quite the same as the `bgRgb` that was here before. The previous RGB value was the _exact_ Nextflow colour and would appear the same on every terminal - irrespective of what theme is being used. The same for the `fgRbg` with (0,0,0).

The problem with terminal colours is that "cyan" and "black" are interpreted differently by everyone's setup, depending on what theme they use. For example, in a dark mode termainal, "black" is actually... white 👀  By using the exact rgb values it always looks the same and it ensures that it'll be readable.

This is minor, but I wanted to point out why I did this particular bit the way I did (and why it was different to everything else).

Reply:
To illustrate this, the output I get now looks like this: (not green and poor text contrast)

![CleanShot 2024-02-10 at 19 46 53@2x](https://github.com/nextflow-io/nextflow/assets/465550/915d6e44-e115-4d76-91de-5bd0c0ea912d)

Whereas before it looked like this:

![CleanShot 2024-02-10 at 19 49 24@2x](https://github.com/nextflow-io/nextflow/assets/465550/d4c49294-f047-44a7-979c-2146e04378f6)

Reply:
Umm, this stuff is little more of a wrapper around an escape code. Considering this is the only place where it's used, is there any chance to find out the code and use it directly?

Reply:
Ok - Yup, can do 👍🏻

Reply:
Ok, got it to work after a bit of trial and error in 32e3a166885285d77dcb7cd3a71058cc345c201b

Now looks exactly as in the second screenshot above.

Reply:
Not working for me. Either we give up with the background color or we may need to use a different library for Ansi colors. [This](https://github.com/dialex/JColor) can be a better choice